client/src/components/Dashboard.js

```diff
@@ -1,40 +1,69 @@
 import React from 'react';
+import { Button, Icon } from 'semantic-ui-react'
+import DashboardMenu from './DashboardMenu.js'
 
 class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            user: {
+                name: 'Gabriel'
+            }
```

Comment:
state value should ideally be a non-mutable data type. How about 

```suggestion
            username: 'Gabriel'
```